recipes/cockroach/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "cockroach" %}
+{% set version = "19.2.5" %}
+
+package:
+  name: go-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://binaries.cockroachdb.com/{{ name }}-v{{ version }}.src.tgz
+  sha256: 7908fcf2dfe9fc525b3a5f60983372ab61dc859f28320bd23b2d9afb77cf31bb
+  patches:
+    - 0001-use-conda-forge-dependencies.patch
+
+build:
+  number: 0
+  #pin_depends: strict
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler("go-cgo") }} >=1.12.17,<1.13
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - cmake >=3.8,<4
+    - protobuf ==3.6.0
```

Comment:
The newer version of "protobuf"generates code go code with warnings, which then break the makefile build. Also, see the comment above on missing headers for libprotobuf.